yolo/json/yolov6.json

```diff
@@ -0,0 +1,108 @@
+{
+    "nn_config":
+    {
+        "output_format" : "detection",
+        "NN_family" : "YOLO",
+        "input_size": "416x416",
+        "NN_specific_metadata" :
+        {
+            "classes" : 80,
+            "coordinates" : 4,
+            "anchors" : [10,13, 16,30, 33,23, 30,61, 62,45, 59,119, 116,90, 156,198, 373,326],
+            "anchor_masks" :
+            {
+                "side52" : [0,1,2],
+                "side26" : [3,4,5],
+                "side13" : [6,7,8]
+            },
+            "iou_threshold" : 0.5,
+            "confidence_threshold" : 0.5
+        }
+    },
+    "mappings":
+    {
+        "labels":
+        [
+            "person",
```

Comment:
label map is the same for v6 too?

Reply:
Yeah because it's trained on COCO so it should be. YoloV5 saves label names in weights, but YoloV6 doesn't. So now this get replaced with [Class_0, Class_1, Class_2, ...] if `num_classes != 80` (COCO).